reactor-error-prone/src/test/resources/com/github/lhotari/reactor/errorprone/testdata/ReactiveStreamsPublisherReturnValueIgnoredPositiveCases.java

```diff
@@ -20,11 +20,6 @@ private static Mono getMono() {
     }
 
     {
-        new Publisher() {
-        };
-        new Flux();
-        new Mono();
```

Comment:
These are not valid constructs. Since they don't have a `// BUG` comment while they do reside in a `*PositiveCases.java` file, I assumed it's okay to delete them. Let me know if you envisioned something else.